org.adempiere.base/src/org/compiere/print/layout/LayoutEngine.java

```diff
@@ -1145,6 +1145,13 @@ else if (item.isTypeBox())			//**	Line/Box
 						element.setMaxWidth(maxWidth);
 					}
 				}
+				/** START DEVCOFFEE: Script print format type **/
+				else if (item.getPrintFormatType().equals("S"))

```

Comment:
comparison should be make against tthe PRINTFORMATTYPE_Script constant instead of the string literal "S" (there are a few occurrence below for the same issue).

Reply:
Fixed in all occurrences.